.lintr

```diff
@@ -0,0 +1,4 @@
+linters: linters_with_defaults(
+    line_length_linter(120),
+    object_length_linter(45)
```

Comment:
Looks good, I can't think of any others we'd want to add at the moment but I like how easy it is to add more.